modules/installation-machines-bare-metal.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-machines-bare-metal-{context}"]
+= Creating the cluster
+
+To create the {product-title} cluster, you provision machines by using the Ingition config
+files that you generated with the installation program.
+
+.Prerequisites
+
+* Your machines must have internet access.
+* Create the required infrastructure for the cluster.
+* You obtained the installation program and generated the Ignition config files
+for your cluster.
+
+.Procedure
+
+. Obtain the {op-system-first} images.
+
+. Use the the ignition config files that you generated by using the installation
+program to create the required bootstrap, worker, and master nodes.
+
+. Monitor for bootstrap-complete
+to monitor cluster bootstrapping.
++
+----
+$ openshift-install --dir test-bare-metal upi bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+----
++
+The command succeeds when the Kubernetes APIServer signals that it has been
+bootstrapped on the control plane machines.
+
+. Optionally, reformat the bootstrap machine.
+
+. Confirm that the cluster recognizes the machines:
++
+----
+$ oc get nodes
+
+NAME      STATUS    ROLES   AGE  VERSION
+master-0  NotReady  master  50s  v1.12.4+c53f462
+master-1  NotReady  master  63s  v1.12.4+c53f462
+master-2  NotReady  master  64s  v1.12.4+c53f462
+worker-0  NotReady  worker  76s  v1.12.4+c53f462
+worker-1  NotReady  worker  70s  v1.12.4+c53f462
+worker-2  NotReady  worker  69s   v1.12.4+c53f462
+----
++
+The output lists all of the machines that you created.
+
+. Review the pending certificate signing requests (CSRs) and ensure that the
+requests are for the machines that you added to the cluster:
++
+----
+$ oc get csr
+----
+
+. Approve the CSRs for your cluster machines. You can approve them individually
+or, if all the CSRs are valid, approve them all by running the following
+command:
++
+----
+$ oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs oc adm certificate approve
+----
+
+. Watch the cluster components come online:
++
+----
+$ watch -n5 oc get clusteroperators
+----
+
+. Provide storage to the image registry.
+
+** For non-production clusters, you
+can set the image registry to an empty directory:
++
+----
+$ oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"storage":{"filesystem":{"volumeSource": {"emptyDir":{}}}}}}'
+----
+
+** For production clusters, you must configure either an Amazon Web Services
```

Comment:
Brandi's working on the modules about configuring registry storage, and [this module](https://github.com/openshift/openshift-docs/pull/14390/files#diff-61d40e8abc6cc529a063b1dd241200e4R1) will need to go into this assembly.